src/content/quickstart/verify_with_discovery.md

```diff
@@ -18,7 +40,7 @@ curl nginx.default.svc.supercluster.local:8080
 
 #### Perform automated verification
 
-This will perform all automated verification between your clusters
+This will perform all automated verification between your clusters.
```

Comment:
Can you elaborate (or link to) more info about quay.io/submariner/nettest  ?
What kind of tests does it run ?

Reply:
we don't have documentation for it right now to link, I have created an issue for it